packages/api-specification/src/window/window-api.md

```diff
@@ -54,15 +54,15 @@ A reference to the new window.
 
 ### Class Methods
 * `addListener(event, callback)` - Add an event listener to execute a given callback
-* `blur()` - Remove focus from the window
-* `close()` - Close the window
-* `focus()` - Give the window focus
-* `hide()` - Hide a visible window (Non browser only)
+* `blur()` - Remove focus from the window. Returns a promise
```

Comment:
You should probably mention what the promise resolves to? Even if it resolves to nothing!